Source/HelixToolkit.SharpDX.Shared/Model/Geometry/MeshGeometry3D.cs

```diff
@@ -192,14 +192,30 @@ public virtual bool HitTest(RenderContext context, Matrix modelMatrix, ref Ray r
                 {
                     int index = 0;
                     float minDistance = float.MaxValue;
+#if !NETFX_CORE
```

Comment:
I don't think this is gonna work.
SharpDX.Core.Wpf project depends on SharpDX.Core. 
You may need to add a separate paramter in HitTest function.

Reply:
Sounds good, I was a bit reluctant to make too many changes, but that seems like the only reasonable way.

Reply:
I couldn't find a way to keep backwards-compatibility and add more parameters so I added a property instead. That might break if the same `MeshGeometry3D` is used in different locations. Is that a likely scenario?

My guess is that the closest hit test will be taken in most cases anyway, then this wont break the calling code, just make it slower.